src/services/contact-notifier.js

```diff
@@ -37,13 +37,13 @@ module.exports = {
     const advertiser = await Advertiser.strictFindById(campaign.advertiserId);
     const subject = `A new campaign was created for ${advertiser.name}`;
     const to = await this.resolveAddresses(campaign.get('notify.internal'));
-    return this.send({ to, subject, html });
+    return to.length ? this.send({ to, subject, html }) : Promise.resolve();
```

Comment:
Since this is an async function the entire contents are already wrapped in a Promise, so there is no need for `Promise.resolve`